blueprint/security/hybrid-system-security-plan.md

```diff
@@ -77,19 +77,20 @@ In accordance with the requirements of the ISM, the following security documenta
 * DTA – Blueprint – Solution Overview (August 2020)
 * DTA – Blueprint – Client Devices Design (August 2020)
 * DTA – Blueprint – Platform Design (August 2020)
-* DTA – Blueprint – Office 365 Design (August 2020) 
-* DTA – Hybrid Blueprint – System Security Plan (October 2020) (this document)
-* DTA – Hybrid Blueprint – System Security Plan Annex (October 2020)
+* DTA – Blueprint – Office 365 Design (August 2020)
+* DTA – Hybrid Blueprint – System Security Plan (April 2021) (this document)
+* DTA – Hybrid Blueprint – System Security Plan Annex (April 2021)
 * DTA – Hybrid Blueprint – Security Risk Management Plan (October 2020)
 * DTA – Hybrid Blueprint – Standard Operating Procedures (October 2020)
 * DTA – Hybrid Blueprint – Incident Response Plan (October 2020)
```

Comment:
Just wondering if these documents are in scope for this review? Similarly for the cloud-native SSP.

Reply:
Yes - will push updates once complete  (for  both Cloud-Native and Hybrid).

Reply:
You should change the date on these too now that you've reviewed them

Reply:
Update pushed